src/mito2/src/sst/parquet/row_group.rs

```diff
@@ -161,16 +165,18 @@ impl<'a> InMemoryRowGroup<'a> {
             // Now we only use cache in dense chunk data.
             self.fetch_pages_from_cache(projection);
 
+            yield_now().await;
```

Comment:
```suggestion
            // Release the CPU to avoid blocking the runtime. Since `fetch_pages_from_cache`
            // is a synchronous, CPU-bound operation.
            yield_now().await;
```